### ISSUE TITLE
How to use Keyboard module

### ISSUE THREAD

**Issue by Richard-Cao** [facebook/react-native#7969]

**Comment by Richard-Cao** [comment]
`Keyboard` module is **undefined** in Android

**Comment by Richard-Cao** [comment]
```
import Keyboard from 'Keyboard';
```
is right but

```
import {Keyboard} from 'react-native';
```
is error

**Comment by adrienthiery** [comment]
Maybe because `Keyboard` is not a module from react-native ?

What are you trying to do ?

**Comment by jsierles** [comment]
The `Keyboard` module will be exported as expected in the upcoming 0.27.1 bugfix release. It's also already fixed in 0.28-rc.
